components/python/jsonschema/history

```diff
@@ -1,4 +1,5 @@
 library/python/jsonschema-34@2.4.0-2020.0.1.3
 library/python-2/jsonschema-26@2.4.0,5.11-2016.0.1.0
 library/python-2/jsonschema@2.4.0,5.11-2016.0.1.0 library/python/jsonschema
-library/python-2/jsonschema-27@2.4.0,5.11-2016.0.1.0 library/python/jsonschema-27
+library/python-2/jsonschema-27@2.4.0,5.11-2016.0.1.0
+library/python/jsonschema-27@2.4.0-2020.0.1.6
```

Comment:
In order to obsolete a package you'll need to increment the COMPONENT_REVISION
library/python/jsonschema-27@2.4.0-2020.0.1.6  ->  library/python/jsonschema-27@2.4.0-2020.0.1.7